sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -3017,6 +3017,291 @@
+                    {
+                      "label": "Upload supporting documents",
+                      "customClass": "font-weight-bold",
+                      "hideLabel": true,
+                      "tableView": false,
+                      "storage": "url",
+                      "dir": "Not a BC Resident - Supporting Documents",
+                      "webcam": false,
+                      "fileTypes": [
+                        {
+                          "label": "",
+                          "value": ""
+                        }
+                      ],
+                      "filePattern": ".pdf,.doc,.docx,.jpg,.png,.txt",
+                      "fileMaxSize": "15MB",
+                      "multiple": true,
+                      "key": "otherLivingSituationDocs1",
+                      "type": "file",
+                      "url": "student/files",
+                      "input": true,
+                      "validate": {
+                        "required": true,
+                        "customMessage": "You must upload at least one file, and the total number of uploaded files cannot exceed {{data.maxUploadedFiles}}.",
+                        "custom": "const files = data[instance.key] || []; valid = files.length > data.maxUploadedFiles ? false : true;"
+                      }
+                    },
+                    {
+                      "label": "Columns",
+                      "columns": [
+                        {
+                          "components": [
+                            {
+                              "html": "<p>We accept <strong>JPG, PNG, DOC, DOCX, PDF, TXT</strong></p>\n",
+                              "label": "Content",
+                              "refreshOnChange": false,
+                              "key": "content48",
+                              "type": "content",
+                              "input": false,
+                              "tableView": false,
+                              "hideOnChildrenHidden": false
+                            }
+                          ],
+                          "width": 6,
+                          "offset": 0,
+                          "push": 0,
+                          "pull": 0,
+                          "size": "md",
+                          "currentWidth": 6
+                        },
+                        {
+                          "components": [
+                            {
+                              "html": "<p style=\"text-align:right;\">15MB file limit each</p>\n",
+                              "label": "Content",
+                              "refreshOnChange": false,
+                              "key": "content49",
+                              "type": "content",
+                              "input": false,
+                              "tableView": false,
+                              "hideOnChildrenHidden": false
+                            }
+                          ],
```

Comment:
There is a simplified version of this content that can be used below.

![image](https://github.com/user-attachments/assets/730f07db-4397-4918-89fd-356edbe03551)

```json
{
  "label": "Content",
  "input": false,
  "className": "file-upload-info-container",
  "content": "<p>We accept <strong>JPG, PNG, DOC, DOCX, PDF, TXT</strong></p>\n<p>15MB file limit each</p>",
  "type": "htmlelement",
  "hideLabel": true,
  "tableView": false
}
```